modules/light-clients/08-wasm/go.mod

```diff
@@ -20,9 +20,9 @@ require (
 	cosmossdk.io/x/evidence v0.1.0
 	cosmossdk.io/x/feegrant v0.1.0
 	cosmossdk.io/x/tx v0.13.1
-	cosmossdk.io/x/upgrade v0.1.0
-	github.com/CosmWasm/wasmvm v1.5.0
-	github.com/cometbft/cometbft v0.38.5
+	cosmossdk.io/x/upgrade v0.1.1
+	github.com/CosmWasm/wasmvm/v2 v2.0.0
+	github.com/cometbft/cometbft v0.38.6
```

Comment:
do we need changelogs for cometbft or x/upgrade? 🤔

Reply:
I can add it for comet.